spec/controllers/users/sessions_controller_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+describe Users::SessionsController, type: :request do
+
+  let (:user) { create_user }
+  let (:login_url) { '/login' }
+  let (:logout_url) { '/logout' }
+
+  context 'When logging in' do
```

Comment:
@josephelias-rubyians  Hi, lets also check for the response data like email, etc